src/Symfony/Component/HttpKernel/EventListener/RouterListener.php

```diff
@@ -135,4 +151,114 @@ public static function getSubscribedEvents()
             KernelEvents::FINISH_REQUEST => array(array('onKernelFinishRequest', 0)),
         );
     }
+
+    private function getWelcomeContent()
+    {
+        $charset = ini_get('default_charset') ?: 'UTF-8';
```

Comment:
The content is fixed, which means its charset also is. This should be also hardcoded.

Reply:
Indeed, thanks.